sharepoint/sharepoint-ps/sharepoint-online/Start-SPOAuditDataCollectionForActivityInsights.md

```diff
@@ -0,0 +1,79 @@
+﻿---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spoauditdatacollectionforactivityinsights
+applicable: SharePoint Online
+title: Start-SPOAuditDataCollectionForActivityInsights
+schema: 2.0.0
+author: pvrk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Start-SPOAuditDataCollectionForActivityInsights
+
+## SYNOPSIS
+
+This cmdlet starts collecting relevant audit data for reports, based on activities within the last 28 days, such as Sharing link reports, content shared with Everyone except external users etc from SharePoint admin center.
+
+## SYNTAX
+
+```
+Start-SPOAuditDataCollectionForActivityInsights -ReportEntity <OptInReportEntityEnum> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet starts collecting relevant audit data for reports, based on activites related to sharing and access. Reports are available for the following scenarios:
+
+- Sharing links created in last 28 days (Anyone, People-in-your-org, Specific people shared externally).
+- Content shared with Everyone except external users (EEEU) in last 28 days.
```

Comment:
>Everyone except external users (EEEU) [](http://example.com/codeflow?start=22&length=37)

what's the official name for this (including casing)?

Reply:
"Everyone except external users" is the official name.

![image](https://github.com/user-attachments/assets/d914cc6c-dbee-444d-aae6-923094572e9b)